build-images.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/env bash
 
-function usage() {
+usage() {
     printf "Usage: $0 [ROS_DISTRO] t]\n"
     printf "Build test and devel images for the provided ROS_DISTRO\n\n"
```

Comment:
With the `-d` option, this is not valid anymore. Please, update the help message or define it back as a positional argument (Personally, I like to have this as a positional argument).

Reply:
Updated 0c0c59d 



> (Personally, I like to have this as a positional argument)

Seems odd use ROS_DISTRO as positional argument here, an then use it as a flag at run-devel.sh